arlo-client/src/types.ts

```diff
@@ -119,4 +125,5 @@ export interface IAudit {
   contests: IContest[]
   jurisdictions: IJurisdiction[]
   rounds: IRound[]
+  errors?: { message: string }[]
```

Comment:
This isn't actually part of an `IAudit`. This would be more properly modeled as `IAudit | IErrorResponse` where you're currently using just `IAudit`.